openmetadata-ui/src/main/resources/ui/src/utils/TimeUtils.ts

```diff
@@ -394,3 +402,10 @@ export const formatDateTimeFromSeconds = (date: number) => {
 
   return dateTime.toLocaleString(DateTime.DATETIME_MED);
 };
+
+/**
+ *
+ * @returns Current date timestamp
+ */
+export const getCurrentDateTimeStampInMilliSeconds = () =>
```

Comment:
```suggestion
export const getCurrentDateStampInMilliSeconds = () =>
```

It's not returning time
Can we stick to mm-dd-yyyy format? and don't make too many formatting methods

Reply:
@chirag-madlani , This method uses another `getTimeStampByDate` which requires date format 'yyyy-MM-dd', which will return the timestamp for the same.

Reply:
As per the discussion, I have removed the `getCurrentDateTimeStampInMilliSeconds` method as we are not using it and marked `getTimeStampByDate` as deprecated.